src/components/Teacher/AddTeacher.js

```diff
@@ -0,0 +1,161 @@
+import {
+  Paper, Snackbar, Alert, TextField,
+} from '@mui/material';
+import Button from '@mui/material/Button';
+import { useEffect, useState, useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { AuthContext } from '../../context/AuthContext';
+import { newTeacher } from '../../features/teachersSlice';
+
+function AddTeacher() {
```

Comment:
- [ ] For some reason, I couldn't add a teacher.  There seems to be a bug that returns unauthorized.

![image](https://user-images.githubusercontent.com/57421823/199835028-746c5583-a5fc-4830-b3f2-ce2551b6c706.png)

This prevented me to delete, or reserving teachers.

Please, correct this bug so your app works smoothly as intended 👍